src/parser/jobs/vpr/modules/SerpentsTail.tsx

```diff
@@ -74,9 +79,10 @@ export class SeprentsTail extends Analyser {
 	}
 
 	private onComplete() {
-		this.checklist.add(new Rule({
+		this.checklist.add(new TieredRule({
 			name: <Trans id = "VPR.serperntstail.wast.name"> Use your <DataLink action="SERPENTS_TAIL"/> follow-ups </Trans>,
 			displayOrder: DISPLAY_ORDER.SEPRENTSTAIL,
+			tiers: TARGET_TIERS,
 			description: <Trans id="vpr.seprentstail.waste.content">
```

Comment:
nitpick: consistency
```suggestion
			description: <Trans id="vpr.serpentstail.waste.content">
```

Reply:
It won't let me add a suggestion, but same suggestion for the Requirement translation tag on line 97.